MathLib/LinAlg/Lis/LisLinearSolver.h

```diff
@@ -65,7 +65,7 @@ class LisLinearSolver final
 
 private:
     LisMatrix& _A;
-    LisOption _option;
+    LisOption _lis_options;
```

Comment:
Minor inconsistency: singular vs. plural forms. I suggest sticking with singular since the class is also called `LisOption`. See also LisEigenLinearSolver.